spec/unit/matrixrtc/MembershipManager.spec.ts

```diff
@@ -938,6 +939,67 @@ describe("MembershipManager", () => {
             expect(client.sendStateEvent).toHaveBeenCalledTimes(0);
         });
     });
+
+    describe("StickyEventMembershipManager", () => {
```

Comment:
This test suite needs much expanding really. This only touches the joins, we really need to test other behaviours.